libs/checkbox/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './lib/checkbox.module';
```

Comment:
You should also export checkbox component to be part of the public API.

Reply:
done